src/encoded/tests/testing_views.py

```diff
@@ -153,7 +153,7 @@ def rev_link_atids(self, request, rev_name):
         "type": "array",
         "items": {
             "type": ['string', 'object'],
-            "linkTo": "TestingLinkSourceSno",
+            "linkTo": "TestingLinkSource",
```

Comment:
What was this change about?

Reply:
This is a bug in the definition that was not throwing a validation error previously but does now with the better validation